src/project.jl

```diff
@@ -14,7 +14,10 @@ function get_path_repo(project::Project, name::String)
     rev = get(source, "rev", nothing)::Union{String, Nothing}
     subdir = get(source, "subdir", nothing)::Union{String, Nothing}
     if path !== nothing && url !== nothing
-        pkgerror("`path` and `url` are conflicting specifications")
+        # When both path and url are specified, path takes precedence (for dev operations)
```

Comment:
This doesn't look right to me, you want the thing entered from the Pkg input to take precedence.